libibumad/man/umad_free_ca_namelist.3

```diff
@@ -0,0 +1,28 @@
+.\" -*- nroff -*-
```

Comment:
New man pages have to be written in markdown

Reply:
All this libibumad code needs some love.

Reply:
New man pages still need to be be in markdown

Reply:
Fixed